contracts/CurrencyNetwork.sol

```diff
@@ -348,6 +353,24 @@ contract CurrencyNetwork is CurrencyNetworkInterface, Authorizable, debtTracking
         );
     }
 
+    /**
+     * @notice `msg.sender` cancels a trustline update it initiated with _debtor
+     * @param _counterparty The other party of the trustline agreement
+     */
+    function cancelTrustlineUpdate(
+        address _counterparty
+    )
+        external
+    {
+        require(! isNetworkFrozen, "The network is frozen; trustlines update cannot be canceled.");
+
+        TrustlineRequest memory trustlineRequest = _loadTrustlineRequest(msg.sender, _counterparty);
+        require(trustlineRequest.initiator == msg.sender, "No update request from msg.sender to _debtor.");
```

Comment:
Maybe also the other party should be able to cancel? (declining a trustline update)

Reply:
Ok